src/megaclient.cpp

```diff
@@ -13278,7 +13326,7 @@ bool MegaClient::syncdown(LocalNode* l, LocalPath& localpath, bool rubbish)
 
             lit++;
         }
-        else if (rubbish && ll->deleted)    // no corresponding remote node: delete local item
+        else if (ll->deleted)    // no corresponding remote node: delete local item
```

Comment:
There is an inner block that test exactly the same again (now that the condition has changed). I'll remove the duplicated tests ;)